test/cmdlineTests/model_checker_solvers_eld/err

```diff
@@ -0,0 +1 @@
+Warning: CHC analysis was not possible since Eldarica was the only Horn solver enabled, but it was not found in the system.
```

Comment:
So this test will fail on all systems that happen to have Eldarica installed :-)?

Reply:
Yes indeed. I think we do the same for cvc4 maybe?

Reply:
Do we? It's suboptimal in any case, but yeah, I'd ultimately not stall this because of that...

Reply:
Yea, most people won't have eld anyway. And this is only a problem for the tests to be 100% precise.

Reply:
We don't even support the newest cvc (cvc5) and haven't used it for ages. I think it's a pragmatic choice to just say we don't use cvc4 in the tests.

Reply:
I mean, I'd see it as pragmatic if there was no other choice but I think that the solution with shadowing `eld` inside `cmdlineTests.sh` is pretty simple and better IMO. Why not do that?

Reply:
Well, we both agree it's a hack, and not even sure that works. boost::process will look for the `eld` binary in the system, so we'd also need to add a wrapper in the system somehow? Also, if we do add a binary that we expect to be called as eldarica, it will influence the expected output, in which case we're basically deciding to always/never have the solver in the system.